frontend/src/app/app-routing.module.ts

```diff
@@ -270,7 +270,7 @@ if (browserWindowEnv && browserWindowEnv.BASE_MODULE === 'liquid') {
   imports: [RouterModule.forRoot(routes, {
     initialNavigation: 'enabledBlocking',
     scrollPositionRestoration: 'enabled',
-    anchorScrolling: 'enabled',
+    anchorScrolling: 'disabled',
```

Comment:
Don't we use this for other components tho @softsimon @mononaut

Reply:
There's the vin/vout anchor links (e.g. https://mempool.space/tx/b00871cec33d359da8d23a0dc6fd685ad776edb5a5d89e8de776ef4e296dc51d#vin=35) but those don't scroll.

There's also the anchor links on the about page but those seem broken to me on mempool.space already and could be fixed to use the mechanism in this PR.